rclcpp/include/rclcpp/time.hpp

```diff
@@ -0,0 +1,75 @@
+// Copyright 2017 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef RCLCPP__TIME_HPP_
+#define RCLCPP__TIME_HPP_
+
+#include <chrono>
+#include <utility>
+
+#include "builtin_interfaces/msg/time.hpp"
+
+#include "rclcpp/exceptions.hpp"
+#include "rclcpp/visibility_control.hpp"
+
+namespace rclcpp
+{
+
+class Time
+{
+  rcl_time_point_value_t rcl_time_;
```

Comment:
Please put member variables at the end of the class definition in its own private section.

If you want the constructors to be private, please mark them explicitly. See: https://google.github.io/styleguide/cppguide.html#Declaration_Order